.rubocop.yml

```diff
@@ -1,92 +1,869 @@
 require:
   - rubocop-performance
+
```

Comment:
Do you think it would be beneficial to extract this out into a general Cerner rubocop config repo that all Ruby based repos can reference?

Reply:
I don't have any objections to that, but I'll wait for someone else to bring that along.